PWGHF/Tasks/taskPidStudies.cxx

```diff
@@ -197,6 +197,7 @@ struct HfTaskPidStudies {
   using CollisionsMc = soa::Join<aod::Collisions, aod::McCollisionLabels, aod::EvSels, aod::CentFT0Cs, aod::CentFT0Ms>;
   using V0sMcRec = soa::Join<aod::V0Datas, aod::V0CoreMCLabels>;
   using CascsMcRec = soa::Join<aod::CascDatas, aod::CascCoreMCLabels>;
+  using BCs = soa::Join<aod::BCsWithTimestamps, aod::BcSels, aod::Run3MatchedToBCSparse>;
```

Comment:
You define in each file this new data type, which looks like to be the same for each file, isn't it? If this is the case, I'd personally put it inside `PWGHF/Utils/utilsEvSelHf.h`

Reply:
I see that in many files (almost everywhere if I am not wrong) you have `using BCs = soa::Join<aod::BCsWithTimestamps, aod::BcSels, aod::Run3MatchedToBCSparse>;`. Since you have always the `#include "PWGHF/Utils/utilsEvSelHf.h"` maybe you can put the `using` there